src/node_errors.cc

```diff
@@ -376,14 +376,7 @@ void AppendExceptionLine(Environment* env,
             .FromMaybe(false));
 }
 
-[[noreturn]] void Abort() {
```

Comment:
Are we replacing it with the macro because we don't want the extra frame here? Which sounds good to me, but just want to confirm.

Reply:
Not exactly, if `Abort` is a function, it must be marked as `[[noreturn]]` to hint compilers that the callee branch would not return so that it will not complain about no return value in the callee. To print a backtrace with `Abort`, the option would be to make it a macro so that the backtrace will not be called in `[[noreturn]] Abort`.

Reply:
hmm, that sounds strange that the compiler complains, because the signature of `Abort` is "returning" void anyway, but I am fine with a macro either way.